### ISSUE TITLE
docker entrypoint initdb?

### ISSUE THREAD

**Issue by deeTEEcee** [diemobiliar/minimized-oraclexe-image#1]
Was wondering what happened to the /docker-entrypoint-initdb.d/startup+setup since that seems to have been removed. Is there a way to add it back in at a later part of the docker image building process?

**Comment by deeTEEcee** [comment]
I think I see how we can add it back in. Will just make a fork of this or something if I get around to it.

**Comment by wemu** [comment]
Hello

Do you refer to some feature of the original oracle image? or do you refer to the Docker Entrypoint? The original Dockerfile https://github.com/oracle/docker-images/blob/main/OracleDatabase/SingleInstance/dockerfiles/18.4.0/Dockerfile.xe#L83 does not contain an ENTRYPOINT statement.

If we need some default data or schema we currently use Flyway or a Test class to set that up. But that may not be a suitable solution for you.

What do you want to use it for? Is that something you can achieve with `docker run` and overwriting the default `CMD` or by using the `--entrypoint` flag?

**Comment by wemu** [comment]
I'm going to close this issue as it seems you have a way for yourselves. If you feel this image would benefit from your changes a PR would be welcome!

**Comment by deeTEEcee** [comment]
Hey, sorry I didn't get to answer this yet. That sounds good to me. 

Here's some feedback after making some changes to the original repo.

> Do you refer to some feature of the original oracle image?

Correct. Actually, similar to other databases (e.g: postgres), there's a way to setup initial fixture data via a script folder with that image:
* /docker-entrypoint-initdb.d/setup
* /docker-entrypoint-initdb.d/startup

I've added some of the original functionality back in https://github.com/deeTEEcee/minimized-oraclexe-image/pull/1 as well as some additional cleanup. You probably don't want my changes in the fork though because I also did excessive changes like removing the aoo_tests user. (I personally didn't see the need to use that and just used a custom superuser instead)
